tests/rpmsigdig.at

```diff
@@ -46,7 +46,17 @@ runroot rpmkeys -Kv /data/RPMS/hello-2.0-1.x86_64.rpm
 [])
 
 RPMTEST_CHECK([
-runroot rpmkeys -Kv /data/RPMS/hello-1.0-1.i386.rpm
+runroot rpmkeys -Kv --nosignature /data/RPMS/hello-2.0-1.x86_64.rpm
```

Comment:
This seems like a duplicated test (?)

Reply:
Hmm, indeed. The idea was to do this test once with --nosignature and once without it, but then I discovered there's another place that already does just that and so this might be just mixed up. I'll double-check.

Reply:
Yup, I had gotten my cases mixed up, fixed now. Good spotting.